package.json

```
@@ -21,16 +21,16 @@
 	"license": "MIT",
 	"author": {
 		"name": "Mario Beltrán Alarcón",
-		"email": "belco90@gmail.com",
+		"email": "me@mario.dev",
 		"url": "https://mario.dev/"
 	},
 	"main": "index.js",
 	"scripts": {
 		"build": "tsc",
 		"postbuild": "cpy README.md ./dist && cpy package.json ./dist && cpy LICENSE ./dist",
-		"docs:gen": "run-p \"docs:gen:*\"",
-		"docs:gen:configs": "ts-node tools/generate-configs",
-		"docs:gen:rules-list": "npm run build && npm run rule-doc-generator && npm run format",
+		"generate:all": "run-p \"gen:*\"",
+		"gen:configs": "ts-node tools/generate-configs",
+		"gen:rules-doc": "npm run build && npm run rule-doc-generator",
```

Maybe make this sub-scripts of `generate`?

```suggestion
		"generate:configs": "ts-node tools/generate-configs",
		"generate:rules-doc": "npm run build && npm run rule-doc-generator",
```

---

I wanted to keep it short, but I don't really care. I'll rename it later so I change the command usages in CI too.